.gitignore

```diff
@@ -0,0 +1,74 @@
+#==============================================================================#
+# File extensions to be ignored anywhere in the tree.
+#==============================================================================#
+# Temp files created by most text editors.
+*~
+# Merge files created by git.
+*.orig
+# Reject files created by patch.
+*.rej
+# Byte compiled python modules.
+*.pyc
+# vim swap files
+.*.sw?
+.sw?
+#OS X specific files.
+.DS_store
+
+# Ignore the user specified CMake presets in subproject directories.
+/*/CMakeUserPresets.json
+
+# Compiled object files
+*.o
+*.mod
+*.a
+*.so
+*.dll
+*.dylib
+*.exe
+*.i
+*.bc
+*.s
+
+
+# Precompiled headers
+*.gch
+*.pch
+
+# Debug files
+*.dSYM/
+*.stackdump
+*.core
+
+# Build directories
+build/
+obj/
+out/
+
+# Generated files
+*.log
+*.tmp
+*.bak
+*.swp
+*.swo
+*.~*
+*.txt
```

Comment:
There are many .txt files and one .log file within aomp that are not generated.  In a clean aomp tree:
[r14 ~/git/aomp21.0/aomp]$ find . -name '*.txt' -print | wc -l
90
[r14 ~/git/aomp21.0/aomp]$ find . -name '*.log' -print | wc -l
1